R/transform.R

```diff
@@ -3,10 +3,11 @@ transform_files <- function(files, transformers) {
     roxy_lines <- get_roxy_lines(text)
 
     new_text <- text
-    new_text[roxy_lines] <- Reduce(
+    text <- paste(text[roxy_lines], collapse = "\n")
```

Comment:
Using the same variable name `text` for a different value looks confusing.

Reply:
Fair enough, we could call it `collapsed_text` or something.